builtin/mainmenu/dlg_create_world.lua

```diff
@@ -78,6 +78,17 @@ local mgv6_biomes = {
 	},
 }
 
+local mapgens_descriptions = {
+	v7 = fgettext("Default mapgen with large complex mountins and plains."),
```

Comment:
```suggestion
	v7 = fgettext("Default mapgen with large complex mountains and plains."),
```
also is it correct to call this default when the game can choose?

Reply:
it's still the engine's default